docs/source/conf.py

```diff
@@ -45,7 +48,7 @@
 
 html_logo = "_static/img/chemicalx.jpg"
 html_static_path = ["_static"]
-html_context = {"css_files": ["_static/css/custom.css"]}
+# html_context = {"css_files": ["_static/css/custom.css"]}
```

Comment:
this seemed to break the build locally but I guess it's working on RTD? I'd rather skip this since local testing is important